DeepLearningSuite/Tools/AutoEvaluator/autoEvaluator.cpp

```diff
@@ -146,15 +148,15 @@ class MyApp:public SampleGenerationApp{
 
                 DetectionsEvaluatorPtr evaluator(new DetectionsEvaluator(reader->getReader(),readerDetection,true));
 
-                evaluator->evaluate(isIouTypeBbox);
-                evaluator->accumulateResults();
+                evaluator->evaluate(isIouTypeBbox, &time_evaluation);
```

Comment:
For both evalute and accumulateResults, store the time in the evaluator class as private attributes, then write methods to return them, this would again help in avoiding to pass any arguments.